errata_tool/cdn_repo.py

```diff
@@ -0,0 +1,96 @@
+from errata_tool import ErrataConnector
+
+
+class CdnRepo(ErrataConnector):
+    """A CdnRepo contains a dict, for example:
+    {
+        'id': 12189,
+        'relationships': {
+            'arch': {
+                'id': 28,
+                'name': 'multi'
+            },
+            'packages': [
+                {
+                    'cdn_repo_package_tags': [
+                        {
+                            'id': 16054,
+                            'tag_template': '{{version}}-{{release}}'
+                        }, {
+                            'id': 16055,
+                            'tag_template': '{{version}}'
+                        }, {
+                            'id': 19195,
+                            'tag_template': 'v{{version(2)}}'
+                        }
+                    ],
+                    'id': 49826,
+                    'name': 'work-container'
+                }
+            ],
+            'variants': [
+                {
+                    'id': 3072,
+                    'name': '8Base-RHACM-2.0'
+                }, {
+                    'id': 3159,
+                    'name': '8Base-RHACM-2.1'
+                }, {
+                    'id': 3284,
+                    'name': '8Base-RHACM-2.2'
+                }, {
+                    'id': 3421,
+                    'name': '8Base-RHACM-2.3'
+                }
+            ]
+        },
+        'type': 'cdn_repos'
+    }
+    """
+
+    def __init__(self, name, product_version, data=None):
+        self.name = name
+        self.data = data
+        self.product_version = product_version
```

Comment:
It's kind of tedious to specify a `product_version` every time. Also, CDN repos can be unassociated from any product version. Let's take this out, and take out `self.url` too since it's so painful.

Reply:
I wasn't so sure about this myself. A url variable might be helpful for conveniently linking a CDN repo for debugging, but I haven't found a way to get a cdn repo URL in the ET web UI without a product version association. I'll drop this at your suggestion.